syntax="proto3";

package fixbug;

option cc_generic_services = true;

message ResultCode{
    int32 errcode=1;
    bytes errmsg=2;
}

message getFriendListRequest{
    uint32 id=1;
}

message getFriendListResponse{
    ResultCode result=1;
    repeated bytes friends=2;
}

service FriendServiceRpc{
    rpc getFriendList(getFriendListRequest) returns(getFriendListResponse);
}